README.md

```diff
@@ -1,2 +1,217 @@
-# Hello_Microverse
-My first repository using Git &amp; GitHub
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./assets/img/Logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello_Microverse project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+**Hello Microverse** is my first project using git & GitHub and I'm very excited to be part of the Microverse community. Also It's my first time using linterns and its configuration.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- Linters
+
+### Tech Stack <a name="tech-stack"></a>
+
+In this project was only used HTML and CSS. More technologies Coming soon.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Semantic HTML**
+- **CSS**
+- **Use of Linters**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://google.com) (Coming soon)
```

Comment:
- [ ] Please kindly delete the live demo section in your readme file since deployment is not part of the project reqirement to deploy the application.